Makefile

```diff
@@ -17,3 +25,7 @@ docker-run:
 
 run:
 	DEBUG=true go run main.go
+
+.PHONY: error
+error:
+	go run github.com/layer5io/meshkit/cmd/errorutil -d . update
```

Comment:
One of them would be updating the `make error` target here
```suggestion
	go run github.com/layer5io/meshkit/cmd/errorutil -d . update -i ./helpers -o ./helpers
```

Reply:
Also, a question for @mgfeller. Are the `errorutil` and `errorutil-analyze` make targets in [meshkit](https://github.com/layer5io/meshkit/blob/8577bc689c39420e58e39967abd1f4957d002d9a/Makefile#L10-L14) a special case or to be general for all the repos? :thinking:

Reply:
@DelusionalOptimist not quite sure what you mean but their are not intended to be a special case, target names can be changed so that they are in sync with the ones in other repos. In `meshkit`, `--skip-dirs meshery` doesn't make sense (not quite sure when this was added). I can fix both when I work on the next errorutil issue in meshkit?

Reply:
> @DelusionalOptimist not quite sure what you mean but their are not intended to be a special case, target names can be changed so that they are in sync with the ones in other repos.

Yes, this was exactly what I wanted to ask. Thanks

> In meshkit, --skip-dirs meshery doesn't make sense (not quite sure when this was added). I can fix both when I work on the next errorutil issue in meshkit?

Umm.. actually I was the on who updated them in https://github.com/layer5io/meshkit/pull/66 on @kumarabd's suggestions. But even I'm not quite sure why :sweat_smile:

Reply:
Hey @DelusionalOptimist I updated the command in the `makefile` and created the `helpers` directory and ran the `make error` command and got this error `ERRO[0000] Unable to execute root command (unknown shorthand flag: 'i' in -i)` Can you help me with this?

Reply:
This is strange. Its working fine for me. :thinking:

Reply:
![Screenshot from 2021-07-11 00-59-30](https://user-images.githubusercontent.com/64970095/125174463-5aac5000-e1e3-11eb-988a-5d2776d96cd7.png)
@DelusionalOptimist you can have a look on this

Reply:
Yes @sayantan1413 taking a look : )

Reply:
So I just cloned your branch and its importing an old version of meshkit. The utility at that version didn't have this flag.
So you'll need to get the latest version of meshkit using,
```
go get github.com/layer5io/meshkit/cmd/errorutil
```

Reply:
Yes @DelusionalOptimist thank you for the help the error got resolved. :)